src/appMain/sdl_preloaded_pt.json

```diff
@@ -1,6 +1,8 @@
 {
     "policy_table": {
         "module_config": {
+            "country_consent_passengersRC": true,

```

Comment:
yes, If we want to extend RC, we will add it later, not now. So please remove un-used code.